cli/main.go

```diff
@@ -17,11 +17,19 @@ Support: support@dispatch.run
 `
 )
 
+var mainCommandText string
```

Comment:
I think this variable doesn't need to be a global, it could be moved into the `createMainCommand` function (or removed if we used `DispatchCmdLong` alone).